recipes/pyode/meta.yaml

```
@@ -0,0 +1,77 @@
+# defining package and version
+{% set name = "pyode" %}
+{% set version = "0.16" %}
+{% set soversion = "0.16.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://bitbucket.org/odedevs/ode/downloads/ode-{{ version }}.tar.gz
+  sha256: 4ba3b76f9c1314160de483b3db92b0569242a07452cbb25b368e75deb3cabf27
+  patches:
+    - add-rt-to-dotpc                # [linux]
+    - modify-dotpc-win               # [win]
+    
+build:
+  number: 0
+  skip: true                         # [win and py==27]
+  
+requirements:
+  build:
+    - {{ compiler('cxx') }}          # stage 1, library
+    - cmake
+    - {{ compiler('c') }}            # stage 2, generate bindings
+    - pkg-config
+    - m2-patch                       # [win]
+    - make                           # [unix]
+
+  host:
+    - cython
+    - python
+    
+outputs:
+  - name: libode
+    requirements:
+      run:
+        - libcxx
+    files:
+      - lib/libode*                                                  # [unix]
+      - lib/pkgconfig                                                # [unix]
+      - lib/cmake                                                    # [unix]
+      - Library/lib/ode_double*                                      # [win]
+      - Library/lib/pkgconfig                                        # [win]
+      - Library/lib/cmake                                            # [win]
+      - include/ode
+    test:
+      commands:
+        - test -f $PREFIX/lib/libode${SHLIB_EXT}                     # [unix]
+        - test -f $PREFIX/lib/libode${SHLIB_EXT}.{{ soversion }}     # [linux]
+        - test -f $PREFIX/lib/libode.{{ soversion }}${SHLIB_EXT}     # [osx]
+        - test -d $PREFIX/include/ode                                # [unix]
+        - test -f $PREFIX/lib/pkgconfig/ode.pc                       # [unix]
+        - if not exist %PREFIX%\Library\lib\ode_double.lib exit 1    # [win]
+        - if not exist %PREFIX%\Library\bin\ode_double.dll exit 1    # [win]
+        - if not exist %PREFIX%\Library\lib\pkgconfig\ode.pc exit 1  # [win]
+  - name: {{ name }}
+    requirements:
+      run:
+        - libode
+    files:
+        - lib/python*/site-packages/*
+    test:
+      imports:
+        - ode
+
+about:
+  home: https://www.ode.org/
+  license: LGPL-2.0 or BSD
```

Which BSD licence is it? (please check I got it right, I only briefly looked)
```suggestion
  license: LGPL-2.0 or BSD-4-Clause
```

---

Counting 4 clauses, so yes.